### PR TITLE
Slovak translation

### PR DESCRIPTION
**Pull request by PatriciaGolianova**
Labels, FAQ, areas

### CODE REVIEW

**Comment by systemcatch** [comment]
@PatriciaGolianova I was able to build your branch, things look ok from a quick pass over.
sv_translation

**Comment by corradio** [comment]
Thanks @PatriciaGolianova !
Were you able to run the system locally?
In order to make the bindings for a new language, you have to add it to the relevant parts of the following section:
https://github.com/tmrowco/electricitymap-contrib/blob/master/web/server.js#L33-L91

**Comment by systemcatch** [comment]
@alixunderplatz ahaha oh wow I should've seen that!

**Comment by PatriciaGolianova** [comment]
Hello guys, 

I have added the required information about new translation to server.js and translation.js. 
But the things I have noticed: 

1. Consumption/Production is not translated - also in Swedish screenshot that @systemcatch madeas well.
snimka obrazovky 2019-01-07 o 11 42 34

2. They are lines that were probably outdated I am not sure about - server.js
snimka obrazovky 2019-01-07 o 11 38 14

Thank you for your patience :)

File: web/server.js [hunk 1]
```diff
@@ -220,6 +222,6 @@ app.all('/dist/*.map', (req, res, next) => {
 app.use(express.static(STATIC_PATH, { etag: true, maxAge: isProduction ? '24h' : '0' }));
 
 // Start the application
-server.listen(process.env['PORT'], () => {
-  console.log(`Listening on *:${process.env['PORT']}`);
+server.listen(8000, () => {
```
Comment (corradio) [inline web/server.js]: Can you revert to the old code?

Comment (PatriciaGolianova) [reply web/server.js]: Fixed in the following commit.

**Comment by corradio** [comment]
thanks!

**Pull request merged**